### ISSUE TITLE
Column reorder issue

### ISSUE THREAD

**Issue by leptitgens** [NadavShaar/react-grid-table#101]
I have problem with the column reorder.

Sometimes, when I move column, I have problem with dispay.
When i moved column, column header stay visible at its initial position, and its also visible in its new position.
The "bad" header remains displayed until the next reload, always at the same place even I scroll or change page. After reload, the column position are ok.

**Comment by NadavShaar** [comment]
Hi, can you please share a live example so i can see the issue?

**Comment by leptitgens** [comment]
I hope this video will be enough :

https://user-images.githubusercontent.com/48163589/196154872-e99bbbc9-2241-49ff-93ac-0cdb99e32e5a.mp4

**Comment by NadavShaar** [comment]
Hi, thank you for your response but unfortunately i can't help you to debug the issue using a video, please try to create a live example in a codesandbox using the same setup you have, thanks.

**Comment by NadavShaar** [comment]
Hi again, any updates?

**Comment by pan-alex** [comment]
Just wanted to comment that I've been experiencing this issue as well. When dragging columns to reorder they may become "stuck" (i.e., the `.rgt-column-sort-ghost` element stays in the DOM after the mouse button is released).

I've experimented with a bit but unfortunately can't reproduce it reliably. I suspect it may have to do with the performance of the application. Here are some of my observations so far:

- The bug seems to occur on Firefox _much_ more frequently than on Chrome/Edge. For the application that I am working: on Firefox, I get a stuck column in <10 re-orders; on Chrome/Edge I seem to be able to do >50 reorders without experiencing the bug; though if I reorder columns _a lot_, I experience the bug on Chrome as well (haven't tried this excessive reordering on Edge).

- For the application that I am working on, I was able to track down a particular commit where the bug became noticeably frequent. I had implemented a button that toggled conditional rendering of the grid-table. Removing the conditional rendering seemed to drastically reduce the frequency of the bug. 
    - However, I couldn't reproduce the issue by adding conditional rendering to a simpler demo app. 
    - When I remove conditional rendering from the current version of my application (which has grown in complexity), the bug is still quite frequent.
